### DISCUSSION TITLE
Visual studio 2022 has workspace facility?

### DISCUSSION THREAD

**Discussion by BhagyashriJaiswal73** [community/community#125424] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Visual studio 2022 has workspace facility?

**Comment by cehfisher** [comment]
Thanks for posting in the GitHub Community, @BhagyashriJaiswal73! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them. I've asked @queenofcorgis to move this issue into the correct category for you. Thank you!
